### PR TITLE
Python3/Plone 5.2 Support

### PR DESCRIPTION
**Pull request by staeff**
Fixes https://github.com/collective/collective.dexteritytextindexer/issues/31 

Needs to be rebased after https://github.com/collective/collective.dexteritytextindexer/pull/32 before it can be merged.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,38 @@
...
-matrix:
-  fast_finish: true
+  - $HOME/buildout-cache
+before_install:
+  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
+  - mkdir $HOME/.buildout
+  - echo "[buildout]" > $HOME/.buildout/default.cfg
+  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
+  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
+  - pip install zc.buildout
```
Comment (gforcada) [inline .travis.yml]: shouldn't that be `pip install -r requirements.txt` ?

File: versions.cfg [hunk 2]
```diff
@@ -1,45 +1,4 @@
...
-pyflakes = 1.5.0
-requests-toolbelt = 0.7.1
-testfixtures = 4.13.4
-twine = 1.8.1
-watchdog = 0.8.3
-z3c.jbot = 0.7.2
-zest.releaser = 6.9
\ No newline at end of file
+setuptools =
+zc.buildout =
```
Comment (gforcada) [inline versions.cfg]: These ones should be the same as requirements.txt.

**Comment by gforcada** [comment]
btw. #32 is already merged, so you could rebase :+1:

File: collective/dexteritytextindexer/converters.py [hunk 3]
```diff
@@ -18,6 +18,7 @@
 from zope.schema.interfaces import ITuple
 
 import logging
+import six
```
Comment (lukasgraf) [inline collective/dexteritytextindexer/converters.py]: I believe `six` should also be declared as a direct dependency of this package in `setup.py`.

File: .travis.yml (hunk 1 above)
Comment (staeff) [reply .travis.yml]: Thanks for the great feedback! I fully agree with you on this one. It was copy and pasted from another add-on and did not see the problem. I'll submit a fix to the other repo as well.

File: versions.cfg (hunk 2 above)
Comment (staeff) [reply versions.cfg]: Yes.

File: collective/dexteritytextindexer/converters.py (hunk 3 above)
Comment (staeff) [reply collective/dexteritytextindexer/converters.py]: Thanks for the review. I agree with your point.

**Pull request merged**